DeepMRSeg/deepmrseg_apply.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python3
```

Comment:
What does this script do? I ran the following command and this is the output:

Singularity> deepmrseg_apply --task dlicv
2021-07-22 14:22:49.813032: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
Task is: dlicv

Reply:
Also, the help is not clear on what args are required for this script.